third_party/move/move-model/src/builder/module_builder.rs

```diff
@@ -1441,7 +1442,7 @@ impl<'env, 'translator> ModuleBuilder<'env, 'translator> {
             let no_mut_ref_param = self.spec_funs[spec_fun_idx]
                 .params
                 .iter()
-                .map(|Parameter(_, ty)| !ty.is_mutable_reference())
+                .map(|Parameter(_, ty, _loc)| !ty.is_mutable_reference())
```

Comment:
```suggestion
                .map(|Parameter(_, ty, _)| !ty.is_mutable_reference())
```

Reply:
Done.

Reply:
I finally got it to update.  For some reason, I seem to have to run "UB=1 git test -p move-prover" in isolation to update the file.

Reply:
What is `git test`? Or perhaps you meant `cargo test`?